### ISSUE TITLE
Some tracks crashed when using AUDACITY Girl in Russian. 

### ISSUE THREAD

**Issue by Taylorcho123** [supertuxkart/stk-code#3333]
## Description

Some tracks crashed when using AUDACITY Girl in Russian. 
AUDACITY girl is an addon kart. 

Tracks which is crashed after loading are as below : 

Standard tracks : 
1. Bovine Barnyard 
2. STK Enterprise
3. Shiny Suburbs

Addon tracks : 
1. BeBeliOuS Circuit 1
2. Old Star Track

## Steps to reproduce

Options -> User Interface -> Russian -> Back to Main manu
-> Singleplayer -> choose AUDACITY Girl 
-> (Race Setup) Select a Novice difficulty and Select a Normal Race mode 
-> and then choose one of the 5 tracks as above.

For your reference, there are also similar bugs in STK when choosing languages which contain non-English characters.

## Configuration

STK release version: 0.9.1

STK Source (ppa, distribution package, official bin,...): from Linux Mint Software Manager.
If you did not download STK from the official website, please specify where you got it:

System: 
OS: Mint 18.3 sylvia
Kernel: x86_64 Linux 4.10.0-38-generic

Graphics card: 
GPU: Mesa DRI Intel(R) Kabylake GT1.5

CPU: Intel Core i5-8250U @ 8x 3.4GHz [42.0°C]

Gamepads/keyboards models if related to the issue:

## Additional information

Please provide stdout.log, it is located in

* %appdata%\supertuxkart\0.8.2 (windows)
* ~/Library/Application Support/supertuxkart/0.8.2 (mac)
* $XDG_CONFIG_HOME/supertuxkart/0.8.2 or ~/.config/supertuxkart/0.8.2 (linux, and other unix based systems)

stdout.log
1. When choosing russian, there was some warnings and errors : 
```
[verbose  ] translation: Env var LANGUAGE = 'ru'.
[verbose  ] translation: Language 'Russian'.
[warn   ] tinygettext: no.po: warning: ignoring, unknown language
[warn   ] tinygettext: no_NO.po: warning: ignoring, unknown language
[warn   ] tinygettext: /usr/share/games/supertuxkart/data/po//ru.po line 22 unknown Plural-Forms given: ""
[warn   ] tinygettext: /usr/share/games/supertuxkart/data/po//ru.po line 2471 msgstr[N] seen, but no Plural-Forms given: ""
[warn   ] tinygettext: /usr/share/games/supertuxkart/data/po//ru.po line 2581 msgstr[N] seen, but no Plural-Forms given: ""
[warn   ] tinygettext: /usr/share/games/supertuxkart/data/po//ru.po line 2590 msgstr[N] seen, but no Plural-Forms given: ""
[error  ] tinygettext: /usr/share/games/supertuxkart/data/po//ru.po line 2744 expected 'msgstr' or 'msgid_plural': ""
[warn   ] tinygettext: /usr/share/games/supertuxkart/data/po//ru.po line 2844 msgstr[N] seen, but no Plural-Forms given: ""
[error  ] tinygettext: /usr/share/games/supertuxkart/data/po//ru.po line 3803 expected empty line: "msgstr "Launchpad Contributions:\nadem https://launchpad.net/~adem4ik\nAlexander 'FONTER' Zinin https://launchpad.net/~spore-09\nAleXoundOS https://launchpad.net/~alexoundos\nAndrei Stepanov, 2015\nAndrey Olykainen https://launchpad.net/~andrey.olykainen\nDenis Deryabin https://launchpad.net/~denis2010der\nDmitriy Koshel https://launchpad.net/~gaim4a\nDmitry Dubrov https://launchpad.net/~dimprogpro\nDmitry Dubrov, 2015\nDmitry https://launchpad.net/~dmitry-ashkadov\nHelen Antonova https://launchpad.net/~vigonett\nIgor Shtompel https://launchpad.net/~i.shtompel\nKirill Babeev https://launchpad.net/~kirill-babeev\nKroArtem https://launchpad.net/~kroartem\nMaxim Kalinin https://launchpad.net/~kalinin-maxim\nOleg, 2015\npingvin https://launchpad.net/~penguin-tux\nSTK-team https://launchpad.net/~stk\nTae-Wong SEO https://launchpad.net/~seotaewong40\nДмитрий https://launchpad.net/~starwars32\nОлег  https://launchpad.net/~laoltomsk\n☠Jay ZDLin☠ https://launchpad.net/~black-buddha666""
```

2. When I choose 'Shiny Suburbs' track : 
```
[verbose  ] RaceManager: Nb of karts=4, ai:3 players:1

[info   ] Singleton: Destroyed singleton.
[fatal  ] STKMesh: Missing texture for material transparent
AL lib: (EE) alc_cleanup: 1 device not closed
```

If your issue is related to the input config (gamepads, keyboards), please provide your file input.xml located in the same directory as stdout.log.

input.xml
```
no keyboard input.

```

If an error message was shown, please copy paste the complete error message or a screenshot of it.

error
```
no error message shown.
```

**Comment by deveee** [comment]
Try STK 0.9.3 first (or even git version if you are able to do it), because STK 0.9.1 won't get any updates and maybe your bug has been fixed some years ago.

**Comment by deveee** [comment]
You can set everything_unlocked=true in config.xml.

There is one stupid translation in russian language that may be related:
```
msgid "%d/%m/%Y"
msgstr "%д.%м.%Г"
```
you can change it to:
```
msgid "%d/%m/%Y"
msgstr "%d/%m/%Y"
```
and see if it helps. At least it will fix date in addons screen. Here is the bug report:
https://github.com/supertuxkart/stk-code/issues/3085

You can also try to remove/rename STK config directory, because maybe there are some incompatibilities with older versions.

And unfortunately I can't reproduce it with STK Enterprise track.

**Comment by Taylorcho123** [comment]
There is  no config.xml in my 0.9.3 github version. I can't find the code such that "msgstr "%д.%м.%Г"" or like this. 

gdb log : 
``
[error  ] PhysicalObject: Unknown shape type : .
[warn   ] PhysicalObject: Uninitialised moving shape
[fatal  ] STKMesh: Missing texture for material transparent
AL lib: (EE) alc_cleanup: 1 device not closed
[Thread 0x7fffe622e700 (LWP 4923) exited]
[Thread 0x7fffe5a2d700 (LWP 4922) exited]
[Thread 0x7fffddec7700 (LWP 4916) exited]
[Thread 0x7fffe420c700 (LWP 4915) exited]
[Thread 0x7fffdece3700 (LWP 4914) exited]
[Thread 0x7fffe6a2f700 (LWP 4910) exited]
[Thread 0x7fffe8ff2700 (LWP 4909) exited]
[Thread 0x7fffe97f3700 (LWP 4908) exited]
[Thread 0x7fffe9ff4700 (LWP 4907) exited]
[Thread 0x7fffea7f5700 (LWP 4906) exited]
[Thread 0x7fffeaff6700 (LWP 4905) exited]
[Thread 0x7fffeb7f7700 (LWP 4904) exited]
[Thread 0x7ffff7fb8140 (LWP 4899) exited]
[Inferior 1 (process 4899) exited with code 01]
(gdb) bt
No stack.
(gdb) 

``

Everytime I make crash in STK and type bt in gdb, there is no stack found.

**Comment by Benau** [comment]
STKMesh: Missing texture for material transparent, you are not using 0.9.3

**Comment by Alayan-stk-2** [comment]
I tried to reproduce.

Results : 
- no crash on the 3 tracks I tested (old star track, bebelious, STK Enterprise)
- the only issue is that the kart is placed too low on the track in the Old star track, apparently because it is too small. However, this issue most likely is in the track itself, and it doesn't depend on language at all (happens with english too)

Though this may exist in 0.9.1, I'm closing this as can't reproduce.
